test/functional/apps/discover/_doc_accessibility.ts

```diff
@@ -52,8 +52,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('should give focus to the first tab link when Tab is pressed', async () => {
-      await browser.pressKeys(browser.keys.TAB);
       const tableTab = await testSubjects.find('docViewerTab-0');
+      await browser.pressKeys(browser.keys.TAB);
```

Comment:
The flaky test runner also caught this one in https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/848, so I put in a fix to wait for the tab to exist before pressing 'Tab'.

Reply:
in German: 2 Fliegen (🪰) mit einer Klappe schlagen  
in English: kill two birds (🐦) with one stone